matrix_client/client.py

```diff
@@ -514,7 +514,7 @@ class User(object):
     """
     def __init__(self, api, user_id):
         if not (user_id.startswith("@") and user_id.count(":") == 1):
-            raise ValueError("UserID's start with @ and seperate with :")
+            raise ValueError("UserID's start with @ and separated with :")
```

Comment:
Yes indeed. In any case, both error messages are grammatically incorrect, so I'd recommend fixing them.